component/file_cache/file_cache_linux_test.go

```diff
@@ -187,6 +178,7 @@ func (suite *fileCacheLinuxTestSuite) TestChmodCase2() {
 	suite.assert.NoError(err)
 
 	// loop until file does not exist - done due to async nature of eviction
```

Comment:
Since eviction is synchronous now, why is the sleep needed?

Reply:
Deletion (where the object is actually being deleted) is synchronous. Eviction is still asynchronous.